src/docs/howto-users-and-authentication/setup-oidc-proxy/index.md

```diff
@@ -78,43 +62,34 @@ at tools like [Ngrok](https://ngrok.com/) or
 [Localtunnel](https://localtunnel.github.io/www/) that allow exposing
 your local ports publicly.
 
```

Comment:
Possible addition:

When serving the template via ngrok, you'll want to make sure the [issuerUrl](https://github.com/sharetribe/web-template/blob/main/server/api-util/idToken.js#L9) matches the URL generated by ngrok where your site is accessible. This might default to localhost if your `NODE_ENV` variable is set to `development`.

If you're using the `yarn run dev-server` command to run the template, this command [overwrites](https://github.com/sharetribe/web-template/blob/main/package.json#L105) the `REACT_APP_MARKETPLACE_ROOT_URL` specified in the .env file. Ensure you update the command to match the URL where your site is being served through ngrok.